setup.py

```diff
@@ -280,9 +282,9 @@ def bazel_build(self, ext):
     if IS_WINDOWS:
       for library_dir in self.library_dirs:
         bazel_argv.append('--linkopt=/LIBPATH:' + library_dir)
-
+    print("303 bazel_build of BuildBazelExtension !!!!!!!!!!!!!!!!!!!!!!!!!!!")
     self.spawn(bazel_argv)
-
+    print("305 bazel_build of BuildBazelExtension !!!!!!!!!!!!!!!!!!!!!!!!!!!")
```

Comment:
nit, clean up this

Reply:
Thanks, Jack, updated in commit: https://github.com/pytorch/xla/pull/5112/commits/cefb6602b317fe37756c8cc7865980bf8744dbf4